lfucg-exactions/client/js/components/AccountLedgersMiniSummary.js

```diff
@@ -0,0 +1,144 @@
+import React from 'react';
+import { Link } from 'react-router';
+import { connect } from 'react-redux';
+import { map } from 'ramda';
+import PropTypes from 'prop-types';
+
+class AccountLedgersMiniSummary extends React.Component {
+    render() {
+        const {
+            currentUser,
+        } = this.props;
+
+        const ledgersList = this.props.mapQualifier && this.props.singleAccountLedger ?
+            (<div>
+                <div className="row link-row">
+                    <div className="col-xs-12 col-sm-5 col-sm-offset-7">
+                        <div className="col-xs-5">
+                            {currentUser && currentUser.permissions && currentUser.permissions.accountledger &&
+                                <Link to={`credit-transfer/form/${this.props.mapSet.id}`} aria-label={`Edit ${this.props.mapSet.entry_date}`}>
+                                    <i className="fa fa-pencil-square link-icon col-xs-4" aria-hidden="true" />
+                                    <div className="col-xs-7 link-label">
+                                        Edit
+                                    </div>
+                                </Link>
+                            }
+                        </div>
+                        <div className="col-xs-5 ">
+                            <Link to={`credit-transfer/summary/${this.props.mapSet.id}`} aria-label={`${this.props.mapSet.entry_date} Summary`}>
+                                <i className="fa fa-file-text link-icon col-xs-4" aria-hidden="true" />
+                                <div className="col-xs-7 link-label">
+                                    Summary
+                                </div>
+                            </Link>
+                        </div>
+                    </div>
+                </div>
+                <div className="col-xs-12">
+                    <p className="col-xs-6">Entry Type: {this.props.mapSet.entry_type_display}</p>
+                    <p className="col-xs-6">Agreement Resolution: {this.props.mapSet.agreement && this.props.mapSet.agreement.resolution_number}</p>
+                    <p className="col-xs-6">Account From: {this.props.mapSet.account_from && this.props.mapSet.account_from.account_name}</p>
+                    <p className="col-xs-6">Account To: {this.props.mapSet.account_to && this.props.mapSet.account_to.account_name}</p>
+                    <p className="col-xs-6">Non-Sewer Credits: {this.props.mapSet.dollar_values && this.props.mapSet.dollar_values.dollar_non_sewer}</p>
+                    <p className="col-xs-6">Sewer Credits: {this.props.mapSet.dollar_values && this.props.mapSet.dollar_values.dollar_sewer}</p>
+                </div>
+            </div>) : (
+                this.props.mapQualifier && map((ledger) => {
+                    return (
```

Comment:
Talked about the need to check the mapQualifier again. I get why we would need to though after talking through my previous comment.